[Docs] 2.4.0 stabilization docs polish pass 2 (#7756)
microsoft/MixedRealityToolkit-Unity: Documentation/Tools/HolographicRemoting.md

> [!NOTE]
> If you cannot connect, make sure your HoloLens 2 is not plugged in to your PC and restart Unity.

## Connecting to the HoloLens with USB cable

> USB cable connection gives better rendering quality and stability. To use USB cable connection, disconnect from Wi-Fi in HoloLens' Settings and launch Holographic Remoting Player app. It will display an IP address starts with 169. Use this IP address in Unity's Holographic Emulation setting to connect. Once you identified the IP address for USB cable, you can connect to Wi-Fi again. 

Reviewer: ```suggestion
USB cable connection gives better rendering quality and stability. To use USB cable connection, disconnect from the HoloLens from Wi-Fi in HoloLens's Settings and launch Holographic Remoting Player app. It will display an IP address that starts with 169. Use this IP address in Unity's Holographic Emulation setting to connect. Once the IP address for USB cable has been identified, it is safe to connect the HoloLens to Wi-Fi again. 
```

Reviewer: Reasons being, as long as we're talking about disconnecting and connecting things from wifi, we should be clear about what device is getting connected/disconnected (in case someone reads quickly and thinks it refers to their laptop, etc)
